pastas/io/base.py

```diff
@@ -118,11 +118,22 @@ def _load_model(data: dict) -> Model:
         n = getattr(ps.noisemodels, data["noisemodel"].pop("class"))()
         ml.add_noisemodel(n)
 
-    # Add fit object to the model
+    # Add solver object to the model from pas-files < 1.3.0  TODO Deprecate
```

Comment:
Is this a future ToDo or a 1.3 ToDo? Perhaps indicate which version number this todo is meant for?

Reply:
That is one to do in 1.4! Such that people can use 1.3 to save to the new format.. So all good.